FirebaseCore/Sources/FIRHeartbeatInfo.m

```diff
@@ -25,9 +27,17 @@ @implementation FIRHeartbeatInfo : NSObject
  */
 + (BOOL)updateIfNeededHeartbeatDateForTag:(NSString *)heartbeatTag {
   @synchronized(self) {
-    NSString *const kHeartbeatStorageFile = @"HEARTBEAT_INFO_STORAGE";
-    GULHeartbeatDateStorage *dataStorage =
-        [[GULHeartbeatDateStorage alloc] initWithFileName:kHeartbeatStorageFile];
+    NSString *const kHeartbeatStorageName = @"HEARTBEAT_INFO_STORAGE";
+    id<GULHeartbeatDateStorable> dataStorage;
+#if TARGET_OS_TV
+    NSString *const kFIRCoreSuiteName = @"com.firebase.core";
```

Comment:
I can  stick with `standardUserDefaults` but felt like I should move  away from it. Thoughts on this domain?

Reply:
A custom domain looks reasonable as there is less probability that it will be modified by developers for their purposes. "com.firebase.core" is pretty generic name, so I would either declare a more specific one for the heartbeat use only or at least define a constant available for the rest of Core if we would like to reuse the same defaults for other purposes. I would prefer the more specific name.

Reply:
I did intentionally name it so Core could use it in the future for other things, since I don't think we need a new suite name for every piece of data we need to store (that's what the keys are for). Breaking it out into a constant would make sense in that case then, and other SDKs could follow suit eventually (`com.firebase.auth`, `.firestore`, etc).

Does that sound reasonable?

Reply:
SGTM. The only potential issue with reuse may is that the same defaults may potentially be modified from different parts of the code that may be easy to overlook. It seems to me that having extra user defaults is a relatively cheap mitigation of this issue. I'm fine with either for this PR.

Reply:
Yeah, we currently use a single user defaults (`standardUserDefaults`) anyways so this wouldn't be anything new, and we should provide a unique name for each usage point.